device_doctor/tool/build.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright 2020 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fetches corresponding dart sdk from CIPD for different platforms, builds
+# an executable binary of device_doctor to `build` folder.
+#
+# This currently supports linux, mac and windows.
```

Comment:
Does this bash script support Windows, or the package?

Reply:
Yes.